### ISSUE TITLE
Cucumber

### ISSUE THREAD

**Issue by lashd** [sky-uk/ruby-bootcamp#21]

**Comment by lashd** [comment]
A bit like with RSpec, a work shop is probably useful here?

Looking for a good video

**Comment by lashd** [comment]
We could get them to add cucumber tests to some of the exercises they do but it is time consuming also so we need to pick which ones. I am thinking the big project, but maybe some of the others too?

**Comment by joshnesbitt** [comment]
I wonder whether the rock, paper, scissors exercise would be good for this? A small chunk of work.

**Comment by lashd** [comment]
Yep, do you think that we should get them do something testing a website? I guess it depends if we want them to have the BDD experience or not?

If we do cucumber against the command line then we should probably do a separate browser testing type exercise? This will definitely be relevant to what they will be doing on going

**Comment by joshnesbitt** [comment]
I think the browser testing is the most valuable. You could test CLI apps with what you learn from the browser testing?

**Comment by lashd** [comment]
Well you would have learnt about the runner irrespective of which type of app you were testing. Browser testing will have certain amount of less transferable learning

**Comment by joshnesbitt** [comment]
I guess so, depends what Sky use the most? I assumed browser but I guess I'm wrong?

**Comment by lashd** [comment]
It absolutely is. I realise now that I may have misread you previous message and like me you are advocating a we based testing exercise.

**Comment by joshnesbitt** [comment]
Yup! Sounds good.

**Comment by lashd** [comment]
I ran a workshop for this the other day am going to try and write it up.

It would be good to identify which exercise we might want to ask people to use it in.

**Comment by joshnesbitt** [comment]
Sounds good!
